setup.py

```diff
@@ -141,7 +141,9 @@ def load_dep_info():
     CXX11_ABI = True
 
 if (cxx11_abi_env_var := os.environ.get("USE_CXX11_ABI")) is not None:
-    if cxx11_abi_env_var == "1":
+    if (
```

Comment:
not sure if this is the right way to do this.

Reply:
Likely this should be handled outside the setup.py so that behavior isnt being masked